.gitignore

```diff
@@ -1,7 +1,187 @@
-node_modules
```

Comment:
I'd prefer we don't make this `.gitignore` change - the previous state was intentionally simple and focused on stuff that matters to us, I'm always happy to add more in to cover the fair needs of contributors, but a lot of the stuff in here is from things that as of right now should never be used in this repo and as far as I'm aware would never be used as a consequence of a contributor's setup (e.g. anything vue, parcel, npm/pnpm/yarn, or dynamodb related)

Can we revert the change here? Let me know if there's a specific item you want added in, but if there is let's just add it into the previous form of this file.

Reply:
This is the default .gitignore you get for node_modules, also for bun init. I think it has sane defaults. I added the ones you had too like `*.pem`. I don't care much about it though. Can revert.